changelog/issue-3779.md

```diff
@@ -0,0 +1,4 @@
+audience: worker-deployers
+level: silent
+reference: issue 3779
+---
```

Comment:
Is it possible existing users are relying on this default?  Perhaps this should come with a note?

Reply:
Good point, I'll add one. Thanks!